tmk_core/protocol/lufa/adafruit_ble_uart.c

```diff
@@ -0,0 +1,377 @@
+#include "adafruit_ble.h"
```

Comment:
The UART BLE code should be a separate PR, and not part of the keyboard PR. 

It should be separate PRs.